docs/src/modules/components/Notifications.js

```diff
@@ -152,23 +156,31 @@ export default function Notifications() {
     <React.Fragment>
       <Tooltip
         open={tooltipOpen}
+        title={t('toggleNotifications')}
+        enterDelay={300}
         onOpen={() => {
           setTooltipOpen(!open);
         }}
         onClose={() => {
           setTooltipOpen(false);
         }}
-        title={t('toggleNotifications')}
-        enterDelay={300}
       >
         <IconButton
           color="primary"
           ref={anchorRef}
           aria-controls={open ? 'notifications-popup' : undefined}
           aria-haspopup="true"
-          onClick={handleToggle}
+          aria-label={`${
+            messageList
+              ? messageList.reduce(
+                  (count, message) => (message.id > lastSeen ? count + 1 : count),
+                  0,
+                )
+              : 0
+          } ${t('toggleNotifications')}`}
```

Comment:
Wrong label "2 Toggle notifications panel" doesn't read well. Fixing this in #40526.